fvm/evm/handler/addressAllocator.go

```diff
@@ -9,7 +9,18 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-const ledgerAddressAllocatorKey = "AddressAllocator"
+const (
+	ledgerAddressAllocatorKey = "AddressAllocator"
+	uint64ByteSize            = 8
+)
+
+var (
+	// prefixes:
+	// the first 12 bytes of addresses allocation
+	// leading zeros helps with storage and all zero is reserved for the EVM precompiles
+	FlowEVMPrecompileAddressPrefix = [...]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
+	FlowEVMCOAAddressPrefix        = [...]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2}
+)
```

Comment:
Given that the prefix is/must be 12 bytes, define a constant and use it here, as well in the related copy calls below:

```suggestion
	addressPrefixLen       = 8
)

var (
	// prefixes:
	// the first 12 bytes of addresses allocation
	// leading zeros helps with storage and all zero is reserved for the EVM precompiles
	FlowEVMPrecompileAddressPrefix = [addressPrefixLen]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
	FlowEVMCOAAddressPrefix        = [addressPrefixLen]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2}
)
```